assets/javascripts/discourse-user-notes/connectors/user-card-post-names/show-user-notes-on-card.js.es6

```diff
@@ -5,7 +5,14 @@ import { emojiUrlFor } from "discourse/lib/text";
 export default {
   shouldRender(args, component) {
     const { siteSettings, currentUser } = component;
-    return siteSettings.user_notes_enabled && currentUser && currentUser.staff;
+    const { user } = args;
```

Comment:
the positioning here feels a tiny bit unsafe... no need to do all this if the site setting is disabled. Perhaps: 

```
if (!(siteSettings.user_notes_enabled && currentUser && currentUser.staff)) {
   return false; 
}

const { user } = args;
return (user && user.user_notes_count > 0) || (user && user.custom_fields && user.custom_fields.user_notes_count >0);

```

This 2 ways of passing the info in though is very awkward. 

```